deploy/kubernetes/overlays/stable/ecr/kustomization.yaml

```diff
@@ -5,4 +5,4 @@ bases:
 images:
   - name: amazon/aws-efs-csi-driver
     newName: 602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/aws-efs-csi-driver
-    newTag: v1.3.3
+    newTag: v1.3.4
```

Comment:
I neglected to link to the newish release process https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/docs/RELEASE.md from somewhere, could you move these kustomize `deploy/` change and the helm `charts/` changes to a separate release-1.3 PR to be merged after the images have been promoted? otherwise users running `kubectl apply -k "github.com/kubernetes-sigs/aws-efs-csi-driver/deploy/kubernetes/overlays/stable/?ref=release-1.3"` might get image pull errors

Reply:
Removed the second commit and stashed the changes for now